packages/rich-text-types/src/nodeTypes.ts

```diff
@@ -157,38 +157,26 @@ export interface EntryHyperlink extends Inline {
 }
 
 export interface TableCell extends Block {
-  nodeType: BLOCKS.TABLE_CELL;
+  nodeType: BLOCKS.TABLE_HEADER_CELL | BLOCKS.TABLE_CELL;
   data: {
     colspan?: number;
     rowspan?: number;
   };
   content: Paragraph[];
 }
 
-export interface TableHeaderCell extends Block {
+export interface TableHeaderCell extends TableCell {
   nodeType: BLOCKS.TABLE_HEADER_CELL;
-  data: {
-    colspan?: number;
-    rowspan?: number;
-  };
-  content: Paragraph[];
 }
 
 export interface TableRow extends Block {
   nodeType: BLOCKS.TABLE_ROW;
   data: EmptyNodeData;
-  content: TableCell[];
-}
-
-// A helper type to only allow all cells in a row to be header cells
-interface TableHeaderRow extends Block {
-  nodeType: BLOCKS.TABLE_ROW;
-  data: EmptyNodeData;
-  content: TableHeaderCell[];
+  content: Array<TableCell>;
```

Comment:
What's different? Just out of curiosity.

Reply:
Actually can be just `TableCell[]`, no difference here